packages/checkout/sdk/src/smartCheckout/balanceCheck/balanceCheck.ts

```diff
@@ -32,6 +32,7 @@ const getTokenBalances = async (
   provider: Web3Provider,
   ownerAddress: string,
   itemRequirements: ItemRequirement[],
+  forceFetch?: boolean,
```

Comment:
Could we just pass a default param so 

`forceFetch: boolean = false`

instead of needing the `!!forceFetch`

Reply:
Good idea, added.